src/kron.jl

```diff
@@ -0,0 +1,66 @@
+export opKron
+
+type opKron{T} <: AbstractLinearOperator{T}
+  nrow :: Int
+  ncol :: Int
+  symmetric :: Bool
+  hermitian :: Bool
+  prod   :: Function
+  tprod  :: Nullable{Function}
+  ctprod :: Nullable{Function}
+end
+
+function opKron(A :: AbstractMatrix,
+                B :: Union{AbstractMatrix, AbstractLinearOperator})
```

Comment:
Now that I think about it, `kron` is an operation that takes linear operators as operands. Perhaps it would be better to extend `kron`, i.e.,
```julia
kron(A::AbstractLinearOperator, B::AbstractLinearOperator) = ... # essentially what you wrote
kron(A::AbstractMatrix, B::AbstractLinearOperator) = kron(LinearOperator(A), B)
kron(A::AbstractLinearOperator, B::AbstractMatrix) = kron(A, LinearOperator(B))
```
What do you think?

Reply:
The way I implemented A needs to be `AbstractMatrix` because I use its elements.